cloud/docker.go

```diff
@@ -93,6 +102,76 @@ func (m *dockerManager) SpawnHost(ctx context.Context, h *host.Host) (*host.Host
 	return h, nil
 }
 
+// attachStdinStream attaches to the container's stdin and streams data to it.
+// Note that the stream to stdin has to run in the background because stdin is
+// not open until the container command is up and running.
+func (m *dockerManager) attachStdinStream(parent *host.Host, container *host.Host) error {
```

Comment:
Worth noting that I stole the general stdin attachment logic from myself when I wrote it [in Jasper](https://github.com/mongodb/jasper/blob/6d9ec4246500a4c9c7d4006d29f053c920e9dc18/internal/executor/docker.go#L152-L158).